README.md

```diff
@@ -332,7 +332,7 @@ sudo make install
 
 ##### Windows
 
-See https://github.com/darktable-org/darktable/blob/master/packaging/windows/BUILD.md
+See https://github.com/darktable-org/darktable/tree/master/packaging/windows
```

Comment:
Probably better to only change BUILD.md to README.md. This way it displays the procedure directly.

Reply:
Actually, the reason it was renamed to README.md was so it was not needed, and the directory is enough - GitHub renders the text then automatically., so I also prefer this link style.

Reply:
@kmilos : I see, I didn't know that the default README.md was read when opening any dir. Works for me then, thanks to both of you.

Reply:
Yep, on GitHub (and GitLab as well?) it is very convenient like that, any README.md/README.txt/README is rendered as the dir page...